app/Services/Backups/InitiateBackupService.php

```diff
@@ -174,4 +172,12 @@ public function handle(Server $server, string $name = null, bool $override = fal
             return $backup;
         });
     }
+
+    public function getNonFailedBackups(Server $server): HasMany
```

Comment:
This should be in a repository if anything, not on a service. It is also called in one place, I'm fine if you just do it right there in the controller.

Reply:
Or even a scope on the model.